src/components/UserSelector.tsx

```diff
@@ -73,10 +67,12 @@ export const UserSelector: React.FC<Props> = ({
               key={user.id}
               href={`#user-${user.id}`}
               onClick={() => {
-                onChange(user);
+                dispatch(setAuthor(user));
+                dispatch(loadPosts(user.id));
+                dispatch(setSelectedPost(null));
               }}
```

Comment:
Pls move the function inside `onClick` handler to a separate const and use as a callback here